gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -1699,4 +1815,52 @@ public static SnapshotListOption fields(SnapshotField... fields) {
    *     Deleting a snapshot</a>
    */
   Operation deleteSnapshot(String snapshot, OperationOption... options);
+
+  /**
+   * Creates a new image.
+   *
+   * @return a global operation for image's creation
+   * @throws ComputeException upon failure
+   */
+  Operation create(ImageInfo image, OperationOption... options);
+
+  /**
+   * Returns the requested image or {@code null} if not found.
+   *
+   * @throws ComputeException upon failure
+   */
+  Image get(ImageId imageId, ImageOption... options);
+
+  /**
+   * Lists images in the provided project that are available to the current user.
+   *
+   * @throws ComputeException upon failure
+   */
+  Page<Image> listImages(String project, ImageListOption... options);
```

Comment:
According to [this](https://cloud.google.com/compute/docs/images#public_images) private images
should be imported to the current project and public images can only be created and maintained by Google.

What I think is more useful is to do what "gcloud compute images list" is doing and return all the publicly available images. 

If that is not doable (provided by the API), we should at least point to a doc  that lists all the Google maintained projects that holds the public images (centos-cloud, debian-cloud, ...).

Also, do you think in this case we should return a functional Image where half of the operations will
fail and the other half are doubtedly useful?